assets/js/util.js

```diff
@@ -118,9 +130,14 @@ $(document).ready(function () {
     });
 
     resizeFooter();
+    var debouncepopulateTranslationList = debounce(populateTranslationList);
+    var debounceresizeFooter = debounce(resizeFooter);
     $(window)
         .on('hashchange', persistChoices)
-        .resize(resizeFooter);
+        .resize(function () {
```

Comment:
Um, are you suggesting something like:

`.resize(debouncepopulateTranslationList)`? and similar for other function too?